### PR TITLE
feat: environment variables for opentelemetry metrics collector endpoint

### PR DESCRIPTION
**Pull request by RogerHYang**
FYI https://horovits.medium.com/prometheus-now-supports-opentelemetry-metrics-83f85878e46a

### CODE REVIEW

File: src/phoenix/config.py [hunk 1]
```diff
@@ -51,12 +51,21 @@
...
+    "PHOENIX_SERVER_INSTRUMENTATION_OTLP_METRIC_COLLECTOR_GRPC_ENDPOINT"
+)
 
 
 def server_instrumentation_is_enabled() -> bool:
     return bool(
         os.getenv(ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_TRACE_COLLECTOR_HTTP_ENDPOINT)
-    ) or bool(os.getenv(ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_TRACE_COLLECTOR_GRPC_ENDPOINT))
+        or os.getenv(ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_TRACE_COLLECTOR_GRPC_ENDPOINT)
+        or os.getenv(ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_METRICS_COLLECTOR_HTTP_ENDPOINT)
```
Comment (mikeldking) [inline src/phoenix/config.py:66]: Isn't it normal to just have a OTLP endpoint and for that to just support metrics too?

File: src/phoenix/server/telemetry.py [hunk 2]
```diff
@@ -1,41 +1,49 @@
...
-            "HTTP collector endpoint should not include the /v1/traces path. Removing it."
-        )
-        # remove the /v1/traces path
-        normalized_endpoint = normalized_endpoint[: -len("/v1/traces")]
-    # remove trailing slashes
-    normalized_endpoint = normalized_endpoint.rstrip("/")
-    return normalized_endpoint
+        endpoint = f"http://{endpoint}"
+    for tail in ("/v1/traces", "/v1/metrics"):
+        if endpoint.endswith(tail):
```
Comment (axiomofjoy) [inline src/phoenix/server/telemetry.py:33]: Do we need to account for trailing slashes?

Comment (RogerHYang) [reply src/phoenix/server/telemetry.py:33]: This is removed on line 24.

Comment (axiomofjoy) [reply src/phoenix/server/telemetry.py:33]: Missed that, thanks.

**Pull request closed without merging**